llama_parse/base.py

```diff
@@ -65,6 +65,10 @@ class LlamaParse(BasePydanticReader):
         default=None,
         description="The API key for the GPT-4o API. Lowers the cost of parsing.",
     )
+    fast_mode: bool = Field(
```

Comment:
Doesn't this conflict with `result_type` ? Now I can set `result_type="markdown"` and `fast_mode=True` and not get markdown?

Reply:
We should remove/deprecated `result_type`? As we do not have the functionality on the API? other idea @logan-markewich ?

Reply:
But there is a result type, it's part of the request url?

And there's still a conflict between that type and this fast mode. Seems like fast mode should just be another result type, like `result_type="text-fast"` 

Otherwise, there would be conflicts between the two params